src/reference/00-Getting-Started/02-Hello.md

```diff
@@ -10,63 +10,64 @@ Hello, World
 
 This page assumes you've [installed sbt][Setup].
 
-### Create a project directory with source code
+### sbt new command
 
-A valid sbt project can be a directory containing a single source file.
-Try creating a directory `hello` with a file `hw.scala`, containing the
-following:
+If you're using sbt 0.13.13 or later, you can use sbt `new` command to quickly setup a simple Hello world build. Type the following command to the terminal.
 
-```scala
-object Hi {
-  def main(args: Array[String]) = println("Hi!")
-}
 ```
+\$ sbt new sbt/scala-seed.g8
+....
+Minimum Scala build.
+
+name [My Something Project]: hello
+
+Template applied in ./hello
+```
+
+When prompted for the project name, type `hello`.
+
+This will create a new project under a directory named `hello`.
 
-Now from inside the `hello` directory, start sbt and type `run` at the sbt
-interactive console. On Linux or OS X the commands might look like this:
+### Running your app
+
+Now from inside the `hello` directory, start `sbt` and type `run` at the sbt shell. On Linux or OS X the commands might look like this:
 
 ```
-\$ mkdir hello
 \$ cd hello
-\$ echo 'object Hi { def main(args: Array[String]) = println("Hi!") }' > hw.scala
 \$ sbt
 ...
 > run
 ...
-Hi!
+[info] Compiling 1 Scala source to /xxx/hello/target/scala-2.12/classes...
+[info] Running example.Hello
+hello
 ```
 
-In this case, sbt works purely by convention. sbt will find the
-following automatically:
-
--   Sources in the base directory
--   Sources in `src/main/scala` or `src/main/java`
--   Tests in `src/test/scala` or `src/test/java`
--   Data files in `src/main/resources` or `src/test/resources`
--   jars in `lib`
-
-By default, sbt will build projects with the same version of Scala used
-to run sbt itself.
+To leave sbt shell, type `exit` or use Ctrl+D (Unix) or Ctrl+Z
+(Windows).
 
-You can run the project with `sbt run` or enter the [Scala
-REPL](http://www.scala-lang.org/node/2097) with `sbt console`. Invoking `sbt console`
-sets up your project's classpath so you can try out live Scala examples
-based on your project's code.
+```
+> exit
+```
 
 ### Build definition
 
-Most projects will need some manual setup. Basic build settings go in a
-file called `build.sbt`, located in the project's base directory.
-
+Basic build settings go in a file called `build.sbt`, located in the project's base directory.
 For example, if your project is in the directory `hello`, in
 `hello/build.sbt` you might write:
 
 ```scala
-lazy val root = (project in file(".")).
-  settings(
-    name := "hello",
-    version := "1.0",
-    scalaVersion := "$example_scala_version$"
+import Dependencies._
+
+lazy val root = (project in file("."))
```

Comment:
> I wonder, could we add some `rootProject` helper to simplify the project in file(".")? 

I think it's possible, but I prefer reducing the number of constructs so it's fewer things to learn, especially if it doesn't add much functionality.

> Also, does it really need to be `lazy`, or could we do without that?

`lazy` does matter because subprojects can refer to each other via `aggregate(...)`, `dependsOn(...)`, and via scoping. You may or may not need it, but "always put `lazy`" is easier than "well it depends.."

> It's also not clear to me what inThisBuild is... does that apply to every project and not just the root project? Can I add it to any subproject and have the same effect? 

`inThisBuild` puts the setting in the build scope, which applies to all subprojects in the build.
Yes, you can add `thisBuild`-scoped settings to any subproject.

> Would it make sense to put these in a separate `val commonSettings = ... `?

`commonSettings` is in fact the way that's [currently](http://www.scala-sbt.org/0.13/docs/Basic-Def.html#How+build.sbt+defines+settings) documented, but it's repeating the same settings scoped to each subproject. `thisBuild`-scoped has a few advantages:
1. Because the value is centralized to a setting, it makes it easier to programmatically change it. This is useful for `version` (e.g. sbt-release uses `version in ThisBuild`)
2. A newly created subproject will be correct by default without manually adding in the common settings.